ports/stm32/boards/WEACT_F411_BLACKPILL/pins.csv

```diff
@@ -0,0 +1,44 @@
+PA0,PA0
+PA1,PA1
+PA2,PA2
+PA3,PA3
+PA4,PA4
+PA5,PA5
+PA6,PA6
+PA7,PA7
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA11,PA11
+PA12,PA12
+PA15,PA15
+PB0,PB0
+PB1,PB1
+PB2,PB2
+PB3,PB3
+PB4,PB4
+PB5,PB5
+PB6,PB6
+PB7,PB7
+PB8,PB8
+PB9,PB9
+PB10,PB10
+PB12,PB12
+PB13,PB13
+PB14,PB14
+PB15,PB15
+PC14,PC14
+PC15,PC15
+LED_BLUE,PC13
+SW,PA0
+SWDIO,PA13
+SWCLK,PA14
+OSC32_IN,PH0
+OSC32_OUT,PH1
+USB_DM,PA11
+USB_DP,PA12
+FLASH_CS,PA4
+FLASH_SCK,PA5
+FLASH_MOSI,PA7
+FLASH_MISO_V2,PB4
+FLASH_MISO_V3,PA6
```

Comment:
Are these flash pins intended to be exposed via board name to the user?  If not then you can do `-FLASH_CS,PA4` (for example); it's still exposed as `pyb_pin_FLASH_CS` but the user won't see `Pin.board.FLASH_CS`.

I don't mind either way (exposing to user or not).

Reply:
There's a few pins here that probably don't really add value for a user, I'd forgotten about the - in this file thanks for the reminder.